src/components/TutorialHeader/TutorialHeader.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Notification20 from '@carbon/icons-react/lib/notification/20';
+import UserAvatar20 from '@carbon/icons-react/lib/user--avatar/20';
+import AppSwitcher20 from '@carbon/icons-react/lib/app-switcher/20';
+
+import {
+  Header,
+  HeaderName,
+  HeaderNavigation,
+  HeaderMenuItem,
+  HeaderGlobalBar,
+  HeaderGlobalAction,
+  SkipToContent,
+} from 'carbon-components-react/lib/components/UIShell';
+
+import { Link } from 'react-router-dom';
+
+const TutorialHeader = () => (
+  <Header aria-label="Carbon Tutorial">
+    <SkipToContent />
+    <HeaderName element={Link} to="/" prefix="IBM">
+      Carbon Tutorial
+    </HeaderName>
+
+    <HeaderNavigation aria-label="Carbon Tutorial">
+      {/* <HeaderMenuItem href="/repos">Repositories</HeaderMenuItem> */}
+      <HeaderMenuItem element={Link} to="/repos">
+        Repositories
+      </HeaderMenuItem>
+
+      <HeaderMenuItem
```

Comment:
You should not have this extra menu link.

Reply:
I have removed extra menu link, please review the changes.